packages/tpetra/core/src/Tpetra_Details_normImpl.hpp

```diff
@@ -252,6 +253,7 @@ gblNormImpl (const RV& normsOut,
     // MPI doesn't allow aliasing of arguments, so we have to make
     // a copy of the local sum.
     RV lclNorms ("MV::normImpl lcl", numVecs);
+    // DEEP_COPY REVIEW - HOST-TO-DEVICE
```

Comment:
Also untested anywhere, and similarly only used in multivector.

Not sure whether this was intended to be used in a wider context than this or not.

Reply:
@MicheldeMessieres Both arguments of this deep_copy have type RV, so the copy should be HOST-TO-HOST or DEVICE-TO-DEVICE, right?   

Also, following the same call chain as for lclNormImpl above, I suspect this code is tested.

Reply:
I confirmed a bunch of Tpetra unit tests exercise this deep_copy.

Reply:
Fixed the comment to device-to-device. I'm calling it 'device' even when it really could be host or device. I'm calling things 'host' only if they are always host.